### PR TITLE
Updated app sidebar

### PR DESCRIPTION
**Pull request by elp49**
1Capture
2Capture
3Capture

### CODE REVIEW

**Comment by elp49** [comment]
Profile sidebar will need to be removed and privacy settings moved to a full page after the privacy settings PR is merged. This would change the Profile and Privacy views

File: frontend/src/components/app/AppHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { useUser } from "../../utils/context"
+
+const useStyles = makeStyles({
+  header: {
+    backgroundColor: "#cd0b2d",
+    height: "3rem",
+    padding: "0 1rem",
+    position: "fixed",
+    left: 0,
+    top: 0,
```
Comment (mneedleman) [inline frontend/src/components/app/AppHeader.tsx]: No unitless css please

**Review by mneedleman — DISMISSED** [review]

File: frontend/src/components/app/AppHeader.tsx (hunk 1 above)
Comment (elp49) [reply frontend/src/components/app/AppHeader.tsx]: this defaults to px

**Review by sb3958 — DISMISSED** [review]

**Review by AyalaCnaan — COMMENTED** [review]
Possibly a stupid question, but should we consider moving the 'contact us' to this sidebar?

**Review by jcarrete5 — DISMISSED** [review]

**Review by jcarrete5 — APPROVED** [review]

**Comment by AyalaCnaan** [comment]
> Possibly a stupid question, but should we consider moving the 'contact us' to this sidebar?

Even if it is a stupid question, I would like an answer.

**Review by jcarrete5 — DISMISSED** [review]

**Comment by elp49** [comment]
> > Possibly a stupid question, but should we consider moving the 'contact us' to this sidebar?
> 
> Even if it is a stupid question, I would like an answer.

yeah the contact page is also in the sidebar

File: frontend/src/components/LoginForm.tsx [hunk 2]
```diff
@@ -60,9 +62,29 @@ export default function LoginForm({
   const history = useHistory()
 
   const { from } = location.state || { from: { pathname: "/" } }
+  const [isOpen, setOpen] = React.useState<boolean>(false)
+  const [type] = useState<NotificationType["type"]>("error")
+  const [message] = useState("Email and/or password entered incorrectly")
```
Comment (jcarrete5) [inline frontend/src/components/LoginForm.tsx]: These states never change and can therefore just be constant.
```suggestion
  const type: NotificationType["type"] = "error"
  const message = "Email and/or password entered incorrectly"
```

File: frontend/src/components/SignupForm.tsx [hunk 3]
```diff
@@ -24,9 +26,36 @@ export default function SignupForm({
   const location = useStatefulLocation()
   const { register } = useUser()
   const { from } = location.state || { from: { pathname: "/" } }
+  const [isModalOpen, setIsModalOpen] = React.useState<boolean>(false)
+  const [isNotifOpen, setNotifOpen] = React.useState<boolean>(false)
+  const [type] = React.useState<NotificationType["type"]>("error")
+  const [message] = React.useState("Something went wrong!")
```
Comment (jcarrete5) [inline frontend/src/components/SignupForm.tsx]: These states never change and can therefore just be constant.
```suggestion
  const type: NotificationType["type"] = "error"
  const message = "Something went wrong!"
```

**Review by jcarrete5 — CHANGES_REQUESTED** [review]

File: frontend/src/components/LoginForm.tsx (hunk 2 above)
Comment (elp49) [reply frontend/src/components/LoginForm.tsx]: this isn't a file I touched, I'm not sure I should make this change in this PR

Comment (jcarrete5) [reply frontend/src/components/LoginForm.tsx]: Weird. I don't know then. The behavior should be the same either way.

**Review by jcarrete5 — DISMISSED** [review]

**Review by jcarrete5 — APPROVED** [review]

**Review by mneedleman — APPROVED** [review]

**Pull request merged**